### ISSUE TITLE
Selecting "Settings" option in dropdown does nothing

### ISSUE THREAD

**Issue by ashucg** [xd4rker/MinerBlock#38]
I am using Firefox Nightly and when I click on Settings, nothing happens and the following errors appear in browser console:

![image](https://user-images.githubusercontent.com/11291284/52895308-6b8e3e00-31b0-11e9-98b4-2f32752ee0f8.png)

**Comment by xd4rker** [comment]
Thanks for reporting this!
Are you on a private window ? If so, this is a known bug on the Firefox version that I plan to fix soon.

**Comment by ashucg** [comment]
> Are you on a private window ? If so, this is a known bug on the Firefox version that I plan to fix soon.

Yeah, I was using private window. I will try it in normal window and let you know if I get any other errors
